tests/ha/firewall_disable.pm

```diff
@@ -14,14 +14,26 @@ use base 'opensusebasetest';
 use strict;
 use testapi;
 use hacluster;
+use version_utils 'is_sle';
+
+sub stop_and_disable_firewall {
+    my $firewall = shift;
+
+    if (is_package_installed "$firewall") {
```

Comment:
I suggest an early return to reduce indentation, e.g.

```perl
return 0 unless is_package_installed ...
assert_script_run ...
```

Reply:
Good point!